cloudy_mozdef/cloudformation/mozdef-instance.yml

```diff
@@ -119,7 +122,8 @@ Resources:
             - content: |
                 OPTIONS_ESSERVERS=${ESURL}
                 OPTIONS_KIBANAURL=${KibanaURL}
-                OPTIONS_METEOR_KIBANAURL=https://relative:9090/_plugin/kibana/
+                OPTIONS_METEOR_KIBANAURL=https://relative:9090/_plugin/kibana/ # not a typo needed for meteor
```

Comment:
I've looked and compared this line that you've added the comment to but can't tell what changed. Was there a missing typo that was re-added? What is this line change about?

Reply:
We should remove my comment line here.  It actually is treated as a string in CF when you do this.  Comments need to be separate lines.

Reply:
@andrewkrug What was the typo you were referring to in the attempted comment?